src/prefect/tasks/airbyte/airbyte.py

```diff
@@ -304,3 +337,112 @@ def run(
             raise FAIL(f"Connection {connection_id} is deprecated.")
+
+
+class AirbyteConfigurationExport(Task):
+    """
+    Task for triggering an export of the Airbyte configuration
+
+    This task assumes that you are using Airbyte Open-Source, since "For
+    Airbyte Open-Source you don't need the API Token for
+    Authentication! All endpoints are possible to access using the
+    API without it."
+    For more information refer to the [Airbyte docs](https://docs.airbyte.io/api-documentation)
+
+    Args:
+        - airbyte_server_host (str, optional): Hostname of Airbyte server where connection is configured.
+            Defaults to localhost.
+        - airbyte_server_port (str, optional): Port that the Airbyte server is listening on.
+            Defaults to 8000.
+        - airbyte_api_version (str, optional): Version of Airbyte API to use to trigger connection sync.
+            Defaults to v1.
+        - **kwargs (Any, optional): additional kwargs to pass to the
+            base Task constructor
+
+    Returns:
+        - compressed gz file byte array of Airbyte configuration data.
+        Airbyte requires this file type (or .tar) for configuration imports
+
+    Example:
+        from prefect import Flow
+        from prefect.tasks.airbyte import AirbyteConfigurationExport
+        import gzip
+
+        airbyte = AirbyteConfigurationExport()
+        with Flow("airbyte_export") as flow:
+            export = airbyte.run()
+            with gzip.open('airbyte.gz', 'wb') as f:
+                f.write(export)
+    """
+
+    def __init__(
+        self,
+        airbyte_server_host: str = "localhost",
+        airbyte_server_port: int = 8000,
+        airbyte_api_version: str = "v1",
+        **kwargs,
+    ):
+        self.airbyte_server_host = airbyte_server_host
+        self.airbyte_server_port = airbyte_server_port
+        self.airbyte_api_version = airbyte_api_version
+        super().__init__(**kwargs)
+
+    def _export_configuration(self, session, airbyte_base_url) -> bytearray:
+        """
+        Trigger an export of Airbyte configuration, see:
+        https://airbyte-public-api-docs.s3.us-east-2.amazonaws.com/rapidoc-api-docs.html#post-/v1/deployment/export
+
+        Args:
+            - session: requests session with which to make call to Airbyte server
+            - airbyte_base_url: URL of Airbyte server
+
+        Returns:
+            - byte array of Airbyte configuration data
+        """
+        get_connection_url = airbyte_base_url + "/deployment/export/"
+
+        try:
+            response = session.post(get_connection_url)
+            if response.status_code == 200:
+                self.logger.debug("Export configuration response: %s", response)
+                export_config = response.content
+                return export_config
+        except RequestException as e:
+            raise AirbyteExportConfigurationFailed(e)
```

Comment:
I recommend including this method as part of the new `AirbyteClient` so that there isn't API call logic spread across multiple classes.

Reply:
done!